app/Repositories/ReportbackRepository.php

```diff
@@ -179,11 +181,47 @@ public function updateReportbackItems($data)
         foreach ($data as $reportbackItem) {
```

Comment:
yes to what dave said! in rogue we'd want multiple reviewers to be able to safely work on the same campaign w/o conflicts